webpack/assets/javascripts/react_app/components/notifications/notifications.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../common/colors.scss';
+
```

Comment:
Yes we do, because the drawer in foreman changes dynamically, and I added a change where the dropdown overlaps the Clear/Read section, which still isn't in PF